tests/Unit/PointwiseFunctions/AnalyticData/GrMhd/Test_KhInstability.cpp

```diff
@@ -0,0 +1,96 @@
+  primitive_variables(const tnsr::I<DataType, 3, Frame::Inertial>& x) const {
+    return this->variables(x, variables_tags<DataType>{});
+  }
+};
+
+template <typename DataType>
+void test(const DataType& used_for_size) {
+  const double adiabatic_index = 1.43;
+  const double strip_bimedian_height = 0.5;
+  const double strip_thickness = 0.4;
+  const double strip_density = 2.1;
+  const double strip_velocity = 0.3;
+  const double background_density = 2.0;
+  const double background_velocity = -0.2;
+  const double pressure = 1.1;
+  const double perturbation_amplitude = 0.1;
+  const double perturbation_width = 0.01;
+  const std::array<double, 3> magnetic_field{{1.0e-3, 0.0, 0.0}};
+  const auto members = std::make_tuple(
+      adiabatic_index, strip_bimedian_height, strip_thickness, strip_density,
+      strip_velocity, background_density, background_velocity, pressure,
+      perturbation_amplitude, perturbation_width, magnetic_field);
+
+  const auto kh_instability = TestHelpers::test_creation<KhInstabilityProxy>(
+      "  AdiabaticIndex: 1.43\n"
+      "  StripBimedianHeight: 0.5\n"
+      "  StripThickness: 0.4\n"
+      "  StripDensity: 2.1\n"
+      "  StripVelocity: 0.3\n"
+      "  BackgroundDensity: 2.0\n"
+      "  BackgroundVelocity: -0.2\n"
+      "  Pressure: 1.1\n"
+      "  PerturbAmplitude: 0.1\n"
+      "  PerturbWidth: 0.01\n"
+      "  MagneticField: [1.0e-3, 0.0, 0.0]\n");
+  CHECK(kh_instability == grmhd::AnalyticData::KhInstability(
+                              adiabatic_index, strip_bimedian_height,
+                              strip_thickness, strip_density, strip_velocity,
+                              background_density, background_velocity, pressure,
+                              perturbation_amplitude, perturbation_width,
+                              magnetic_field));
+
+  KhInstabilityProxy kh_inst_to_move(
+      adiabatic_index, strip_bimedian_height, strip_thickness, strip_density,
+      strip_velocity, background_density, background_velocity, pressure,
+      perturbation_amplitude, perturbation_width, magnetic_field);
+  test_move_semantics(std::move(kh_inst_to_move), kh_instability);  //  NOLINT
+
+  // run post-serialized state through checks with random numbers
+  pypp::check_with_random_values<1>(
+      &KhInstabilityProxy::template primitive_variables<DataType>,
+      serialize_and_deserialize(kh_instability), "KhInstability",
+      {"rest_mass_density", "velocity", "specific_internal_energy", "pressure",
+       "lorentz_factor", "specific_enthalpy", "magnetic_field",
+       "divergence_cleaning_field"},
+      {{{0.0, 1.0}}}, members, used_for_size);
+}
+}  // namespace
+
+SPECTRE_TEST_CASE("Unit.PointwiseFunctions.AnalyticData.GrMhd.KhInstability",
+                  "[Unit][PointwiseFunctions]") {
+  pypp::SetupLocalPythonEnvironment local_python_env{
+      "PointwiseFunctions/AnalyticData/GrMhd"};
+
+  test(std::numeric_limits<double>::signaling_NaN());
+  // test(DataVector(5));
```

Comment:
Test for `DataVector` was skipped?

Reply:
Left over from debugging. Thanks for catching :)